cli/server/server.go

```diff
@@ -491,7 +491,7 @@ func startServer(ctx *cli.Context) error {
 
 	go serv.Start(errChan)
 	if !cfg.ApplicationConfiguration.RPC.StartWhenSynchronized {
-		rpcServer.Start()
+		go rpcServer.Start()
```

Comment:
Could you add a comment to the code, please?

Reply:
added, pls check